js/flightlog_fields_presenter.js

```diff
@@ -1056,7 +1088,16 @@ function FlightLogFieldPresenter() {
                             return value.toFixed(0) + "Hz";
                     }
                 case 'GPS_RESCUE_THROTTLE_PID':
-                        return value.toFixed(0);
+                    switch (fieldName) {
+                        case 'debug[0]': // Throttle P added uS
+                        case 'debug[1]': // Throttle D added * uS
+                            return value.toFixed(0) + 'uS';
```

Comment:
the `uS` unit here is different from line 1031 or 1035, where is lowercase. I think all of them must be equal.
```suggestion
                    switch (fieldName) {
                        case 'debug[0]': // Throttle P added us
                        case 'debug[1]': // Throttle D added * us
                            return value.toFixed(0) + 'us';
```

Reply:
thanks, done